docker-entrypoint.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+# Check whether there is a passwd entry for the container UID
+myuid=$(id -u)
+mygid=$(id -g)
+uidentry=$(getent passwd $myuid)
+
+# If there is no passwd entry for the container UID, attempt to create one
+if [ -z "$uidentry" ] ; then
+    if [ -w /etc/passwd ] ; then
+        echo "$myuid:x:$myuid:$mygid:anonymous uid:/home/user:/bin/false" >> /etc/passwd
+    else
+        echo "Container ENTRYPOINT failed to add passwd entry for anonymous UID"
+    fi
+fi
+kebechet run
```

Comment:
shouldnt this be some `exec "$@"` and the Dockerfile like

```
ENTRYPOINT [ "kebechet" ]
CMD run
```

Reply:
```exec "$@"``` means to execute all positional parameter. right?
In that case should be: 
```
ENTRYPOINT ["/docker-entry.sh"] (the executable file)
CMD ["kebechet","run"](positional parameters!?)
```
But the confusion to me is that kebechet is an executable command, whereas the run is a parameter to kebechet. Adding ```exec "$@"``` confuses me.